prijateli_tree/app/routers/games.py

```diff
@@ -411,32 +431,7 @@ def score_to_denirs(
     return {"reward": f"You have made {denirs} denirs!"}
 
 
-@router.post("/{game_id}/player/{player_id}/integrated")
-def integrated_game(
-    game_id: int, player_id: int, db: Session = Depends(get_db)
-):
-    """
-    Logic for handling the integrated game
-    """
-    game, player = get_game_and_player(game_id, player_id, db)
-
-    # Get current round
-    current_round = get_current_round(game_id, db)
-
-    if current_round > game.rounds:
-        return {"message": "Game over"}
-
-    view_round(game_id, player_id, db)
-    # Update the player's answer if they want to - HOW?!
-    route_add_answer(game_id, player_id, "", current_round, db)
-    # Calculate the score
-    route_add_score(game_id, player_id, db)
-    if current_round == game.rounds:
-        # Final round - calculate the denirs
-        score_to_denirs(game_id, player_id, db, current_round)
-
-
-@router.post("/ready")
+@router.post("/player_ready")
```

Comment:
Since the `player_id` is included in the payload, I'd prefer this stay as `ready`.